playbilling/src/main/java/com/google/androidbrowserhelper/playbilling/digitalgoods/AcknowledgeCall.java

```diff
@@ -0,0 +1,54 @@
+package com.google.androidbrowserhelper.playbilling.digitalgoods;
+
+import android.os.Bundle;
+
+import androidx.annotation.Nullable;
+
+/**
+ * A class for parsing Digital Goods API calls from the browser and converting them into a format
+ * suitable for calling the Play Biling library.
+ */
+public class AcknowledgeCall {
```

Comment:
This may be overkill, so feel free to ignore,  but the similarity between `AcknowledgeCall` and `GetDetailsCall` make me feel like they have a common shared interface. 

eg:
```java
interface BillingCall<T> {
    void respond(int responseCode, T details) {
}
```

then 

```java
class AcknowledgeCall implements BillingCall<Void>
```
and 

```java
class GetDetailsCall implements BilllingCall<ItemDetails[]>
```

Reply:
Yeah, I was considering that, but ultimately I couldn't see where we would use that interface. If an opportunity becomes apparent we can add it.